en/mozillavpn.xliff

```diff
@@ -0,0 +1,556 @@
+      <source xml:space="preserve">Terms of Service</source>
+      <context-group purpose="location"><context context-type="linenumber">20</context></context-group>
+    </trans-unit>
+    <trans-unit id="vpn.aboutUs.privacyPolicy">
+      <source xml:space="preserve">Privacy Policy</source>
+      <context-group purpose="location"><context context-type="linenumber">26</context></context-group>
+    </trans-unit>
+    <trans-unit id="vpn.settings.aboutUs">
+      <source xml:space="preserve">About us</source>
+      <context-group purpose="location"><context context-type="linenumber">36</context></context-group>
+      <context-group purpose="location"><context context-type="sourcefile">../src/ui/settings/ViewSettingsMenu.qml</context><context context-type="linenumber">49</context></context-group>
+    </trans-unit>
+    <trans-unit id="vpn.main.productDescription">
+      <source xml:space="preserve">A fast, secure and easy to use VPN. Built by the makers of Firefox.</source>
+      <context-group purpose="location"><context context-type="linenumber">62</context></context-group>
+      <context-group purpose="location"><context context-type="sourcefile">../src/ui/views/ViewInitialize.qml</context><context context-type="linenumber">33</context></context-group>
+      <context-group purpose="location"><context context-type="sourcefile">../src/main.cpp</context><context context-type="linenumber">60</context></context-group>
+    </trans-unit>
+    <trans-unit id="vpn.aboutUs.releaseVersion">
+      <source xml:space="preserve">Release Version</source>
+      <context-group purpose="location"><context context-type="linenumber">71</context></context-group>
+    </trans-unit>
+  </body></file>
+  <file datatype="plaintext" original="../src/ui/settings/ViewLanguage.qml" source-language="en" target-language="en-US"><body>
+    <trans-unit id="vpn.settings.language">
+      <source xml:space="preserve">Language</source>
+      <context-group purpose="location"><context context-type="linenumber">20</context></context-group>
+      <context-group purpose="location"><context context-type="sourcefile">../src/ui/settings/ViewSettingsMenu.qml</context><context context-type="linenumber">42</context></context-group>
+    </trans-unit>
+    <trans-unit id="vpn.settings.system">
+      <source xml:space="preserve">System</source>
+      <context-group purpose="location"><context context-type="linenumber">46</context></context-group>
+      <note annotates="source" from="developer">The system language</note>
+    </trans-unit>
+    <trans-unit id="vpn.settings.languageAccessibleName">
+      <source xml:space="preserve">%1 %2</source>
+      <context-group purpose="location"><context context-type="linenumber">64</context></context-group>
+      <context-group purpose="location"><context context-type="linenumber">116</context></context-group>
+    </trans-unit>
+    <trans-unit id="vpn.settings.additional">
+      <source xml:space="preserve">Additional</source>
```

Comment:
Is there a screenshot of how this string is used?

Reply:
Of course. I'll add a note that this refers to additional Languages.

![Screenshot 2020-10-28 at 18 50 08](https://user-images.githubusercontent.com/9611612/97476250-7c63f080-194e-11eb-8925-4669d6317f44.png)